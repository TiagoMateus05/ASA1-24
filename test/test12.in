5 10
3 2 5 3 3
1 4 4 1 1
3 4 5 1 5
2 4 2 5 2
3 4 3 2 4
4 1 3 5 3 2 4 4 5 1
2