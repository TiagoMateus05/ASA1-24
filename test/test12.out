1
(((((((4 1) 3) 5) (3 (2 4))) 4) 5) 1)
